src/lib.rs

```diff
@@ -75,7 +75,32 @@ use tracing;
 /// The client returned by `connect` function
 ///
 /// This is a convenience type which you most likely want to use instead of raw client.
-pub type Client = rpc::lightning_client::LightningClient<tonic::codegen::InterceptedService<tonic::transport::Channel, MacaroonInterceptor>>;
+pub struct Client {
+    lightning: rpc::lightning_client::LightningClient<tonic::codegen::InterceptedService<tonic::transport::Channel, MacaroonInterceptor>>,
+    wallet: wallet::wallet_kit_client::WalletKitClient<tonic::codegen::InterceptedService<tonic::transport::Channel, MacaroonInterceptor>>,
+}
+use tonic::IntoRequest;
+
+impl Client {
+    
+    pub fn lightning(&self) -> &rpc::lightning_client::LightningClient<tonic::codegen::InterceptedService<tonic::transport::Channel, MacaroonInterceptor>> {
+        &self.lightning
+    }
```

Comment:
Once the methods are inherited, I think this should be represented privately as `inner`